src/main/java/io/jenkins/plugins/artifact_manager_s3/S3BlobStore.java

```diff
@@ -199,14 +196,28 @@ public URL toExternalURL(@NonNull Blob blob, @NonNull HttpMethod httpMethod) thr
         return builder.build().generatePresignedUrl(container, name, expiration, awsMethod);
     }
 
+    public boolean isConfigured(){
```

Comment:
What happens if the user tries to run a build without a container set? Is there a clear error?